### PR TITLE
Library update notices

### PR DESCRIPTION
**Pull request by andrewconner**
@cvializ Whatcha think? After roughly an hour of no updates, it stops checking, and then lets you do an elegant `$state.reload()`

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/library.js [hunk 1]
```diff
@@ -560,6 +562,59 @@ angular.module('kifi')
       });
     });
 
+    var deregisterUpdateLibrary;
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/libraries/library.js:565]: If we like this, we could turn this into a utility function to use elsewhere. Most of the logic is generic and boilerplate.

File: kifi-backend/modules/shoebox/angular/src/keeps/keeps.styl [hunk 2]
```diff
@@ -22,6 +22,17 @@ maxWidthForNarrowKeep = keepCardMaxWidth + 2 * (keepSidePad + appSidePad1) - 1 /
...
+.kf-keeps-update
+  display: inline-block
+  padding: 8px 24px
+  border: 1px solid #f8f4e8
+  border-radius: 2px
+  font-size: 16px
+  font-weight: 300
+  background-color: #ffc
+  white-space: nowrap
+  position: relative
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/keeps/keeps.styl]: I don't think `position: relative` does anything here. It's only use cases are when 1. an element contains `position: absolute` children, or 2. the element itself has `top`,`right`,`bottom`,`left` properties. I don't see those conditions here

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/keeps/keeps.styl]: You're right, it's there from when I was positioning it differently.

**Comment by cvializ** [comment]
:tada: lgtm

**Pull request merged**